src/status_im/chat/models.cljs

```diff
@@ -12,10 +13,10 @@
   (update-in db [:chat-ui-props current-chat-id ui-element] not))
 
 (defn- create-new-chat
-  [{:keys [db gfy-generator now]} chat-id chat-props]
-  (let [{:keys [name whisper-identity]} (get-in db [:contacts/contacts chat-id])]
+  [{:keys [db now]} chat-id chat-props]
+  (let [name (get-in db [:contacts/contacts chat-id :name])]
     (merge {:chat-id    chat-id
-            :name       (or name (gfy-generator whisper-identity))
```

Comment:
Does this mean we can remove `gfy-generator` fn? If not, why do we have `gfy-generator` and `generate-gfy`?

Reply:
it's not a function it's a co-effect and it wasn't  even in the list of coeffects passed to the function

Reply:
Right, I'm referring to the coeffect which has a function. Do we need both?

Reply:
it's used somewhere else. should be removed IMO and replaced by the function but not in this PR

Reply:
OK

Reply:
Can you add this as a TODO above the cofx?